webpack/assets/javascripts/react_app/components/Layout/components/TaxonomySwitcher/TaxonomyDropdown.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { ContextSelectorItem } from '@patternfly/react-core';
+import CustomContextSelector from '../CustomContextSelector';
+import { noop } from '../../../../common/helpers';
+import { translate as __ } from '../../../../common/I18n';
+
+const TaxonomyDropdown = ({
+  taxonomyType,
+  currentTaxonomy,
+  taxonomies,
+  changeTaxonomy,
+  anyTaxonomyText,
+  manageTaxonomyText,
+  anyTaxonomyURL,
+  manageTaxonomyURL,
+  ...props
+}) => {
+  const [searchValue, setSearchValue] = useState('');
+  const [isOpen, setIsOpen] = useState(false);
+  const [filteredItems, setFilteredItems] = useState(taxonomies);
+
+  const onToggle = (event, newIsOpen) => {
+    setIsOpen(newIsOpen);
+  };
+  const onSelect = () => {
+    setIsOpen(!isOpen);
+  };
+  const onSearchInputChange = (value, event) => {
+    setSearchValue(event.target.value);
+    onSearchButtonClick();
```

Comment:
the filter doesn't work good as earlier, in some cases, it doesn't show the correct values when the input change :(

Reply:
In the PF demo they have this code but without ` onSearchButtonClick();` so it only searches when a user clicks. I think I'll change to that so it will be simpler, what do you think?

Reply:
I think that for now if we filter upon input change, we should keep doing so,
though if things are not working well, we could stick with the PF4 implementation and open an issue there.

Reply:
This is not a PF4 issue, apparently the useState set isn't immediate, so I work around that :)